Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'charlock_holmes', '~> 0.6.9.4'
 gem 'dynamic_form', '~> 1.1.4'
 gem 'exception_notification', '~> 3.0.1'
 gem 'fancybox-rails', '~> 0.2.1'
-gem 'foundation-rails', '~> 5.2.1.0'
+gem 'foundation-rails', '~> 5.5.3.2'
```

Comment:
Latest version of `foundation-rails` is [6.2.1.0](https://rubygems.org/gems/foundation-rails). Worth adding notes to the commit message about why can't we go further.